### PR TITLE
CEVIEW review take 2

### PR DESCRIPTION
**Pull request by brianhlin**
@djw8605 This is a WIP but figured I'd get this up sooner rather than later so you can take a look at your convenience.

### CODE REVIEW

File: config/htcondor-ce.spec [hunk 1]
```diff
@@ -277,10 +277,10 @@ fi
 %defattr(-,root,root,-)
 
 # Web package
-%{python_sitelib}/htcondor_ce
-%{python_sitelib}/htcondor_ce/__init__.py
-%{python_sitelib}/htcondor_ce/web.py
-%{python_sitelib}/htcondor_ce/rrd.py
```
Comment (djw8605) [inline config/htcondor-ce.spec]: Why are you including the `*` after the .py?  Rpmbuild will automatically add the pyc when it precompiles the python files.

File: config/htcondor-ce.spec [hunk 2]
```diff
@@ -277,10 +277,10 @@ fi
...
+%{python_sitelib}/htcondorce
+%{python_sitelib}/htcondorce/__init__.py*
+%{python_sitelib}/htcondorce/web.py*
+%{python_sitelib}/htcondorce/rrd.py*
```
Comment (djw8605) [inline config/htcondor-ce.spec]: Why are you including the * after the .py? Rpmbuild will automatically add the pyc when it precompiles the python files.

Comment (brianhlin) [inline config/htcondor-ce.spec]: Oh, I didn't know that! Fixed!

**Comment by djw8605** [comment]
So far, everything seems good and reasonable.   Do you have much more?  I can do some final testing once you are done.

**Comment by brianhlin** [comment]
I think that's all I've got for now.

**Comment by djw8605** [comment]
@brianhlin The rpmbuild fails because it is unable to find bdii related files:
```
RPM build errors:
    File not found: /home/centos/rpmbuild/BUILDROOT/htcondor-ce-1.20-2.el6.x86_64/var/lib/bdii/gip/provider/condor_ce_bdii_generate_glue1.py
    File not found: /home/centos/rpmbuild/BUILDROOT/htcondor-ce-1.20-2.el6.x86_64/var/lib/bdii/gip/provider/condor_ce_bdii_generate_glue2.py
    File not found: /home/centos/rpmbuild/BUILDROOT/htcondor-ce-1.20-2.el6.x86_64/usr/share/condor-ce/config.d/06-ce-bdii-defaults.conf
    File not found: /home/centos/rpmbuild/BUILDROOT/htcondor-ce-1.20-2.el6.x86_64/etc/condor-ce/config.d/06-ce-bdii.conf
```

**Comment by brianhlin** [comment]
@djw8605 should be done now

**Comment by djw8605** [comment]
Now I'm getting these build errors:
```
RPM build errors:
    File listed twice: /usr/lib/python2.6/site-packages/htcondorce/__init__.py
    File listed twice: /usr/lib/python2.6/site-packages/htcondorce/info_query.py
    File listed twice: /usr/lib/python2.6/site-packages/htcondorce/tools.py
    Installed (but unpackaged) file(s) found:
   /etc/condor-ce/config.d/06-ce-bdii.conf
   /usr/share/condor-ce/config.d/06-ce-bdii-defaults.conf
```

**Comment by brianhlin** [comment]
@djw8605 I forgot to open the file as writeable. I think that should solve our problem.

**Comment by djw8605** [comment]
Yup, the view seems to work now.  Thanks!
image

**Pull request merged**